.azure-pipelines/generation-pipeline.yml

```diff
@@ -517,7 +517,6 @@ stages:
             branchName: 'kiota/$(betaBranch)'
             targetClassName: "BaseGraphClient"
             targetNamespace: 'Microsoft\\Graph\\Beta\\Generated'
-            baseBranchName: 'dev'
```

Comment:
there's no default for the parameter, either set a default or restore those with main as a value

Reply:
Updated.